warehouse/query-core/src/main/java/datawave/query/jexl/visitors/ExecutableDeterminationVisitor.java

```diff
@@ -717,55 +714,7 @@ public Object visit(ASTNENode node, Object data) {
 
     @Override
     public Object visit(ASTReference node, Object data) {
```

Comment:
Do we need to keep a visit method for the ASTReference ?

Reply:
In the previous code we checked for markers at the reference node - that logic has been added to the and node method.  There was no other special functionality for reference nodes, so unless there is some special functionality we want to run for these nodes, I think the base visitor method is fine.